pytorch/training/buildspec-2-2-ec2.yml

```diff
@@ -5,16 +5,16 @@ framework: &FRAMEWORK pytorch
 version: &VERSION 2.2.0
```

Comment:
any reason why we're only modifying ec2 buildspec?

Reply:
Release for both SM and EC2